### ISSUE TITLE
Make lists in site more accessible for keyboard navigation

### ISSUE THREAD

**Issue by aniri** [code4romania/ce-ma-fac#204]
The following issues from the components repo should be fixed:
- https://github.com/code4romania/taskforce-fe-components/issues/170
- https://github.com/code4romania/taskforce-fe-components/issues/169
- https://github.com/code4romania/taskforce-fe-components/issues/168

Then the components version will need an update here :smile:
